aptos-move/framework/aptos-framework/sources/configs/consensus_config.spec.move

```diff
@@ -12,7 +12,7 @@ spec aptos_framework::consensus_config {
         aborts_if !system_addresses::is_aptos_framework_address(addr);
         aborts_if exists<ConsensusConfig>(@aptos_framework);
         aborts_if !(len(config) > 0);
-        ensures  exists<ConsensusConfig>(@aptos_framework);
+        ensures global<ConsensusConfig>(addr) == ConsensusConfig { config };
```

Comment:
keep exists

Reply:
well is L15 has already cover the "ensures  exists<ConsensusConfig>(@aptos_framework);" ?